packages/addon-mcp/README.md

```diff
@@ -18,7 +15,7 @@ The addon provides tools to improve agents' UI development capabilities, retriev
 ### Installation and Setup
 
 > [!NOTE]
-> This addon requires Storybook 9.0 or higher.
+> This addon requires Storybook 10.1 or higher, currently only available as the canary verison `0.0.0-pr-32816-sha-28f59f53`.
```

Comment:
we'll update this version string everywhere whenever the PR to core has been merged and released.